index.js

```diff
@@ -25,9 +25,15 @@ var EDGEMAP = {
 util.inherits( FacebookInsightStream, stream.Readable )
 function FacebookInsightStream( options ) {
     stream.Readable.call( this, { objectMode: true } );
+    var isFunction = typeof options.itemList === 'function';
+    if ( !isFunction ) {
+        var itemList = options.itemList;
+        options.itemList = function () { return itemList }
+    }
 
     options.edge = EDGEMAP[ options.node ];
     this.options = options;
+    console.log( options.itemList )
```

Comment:
Also - is this logging a func definition? 

`options.itemList = function () { return itemList }`